README.md

```diff
@@ -1,138 +1,192 @@
 
-#### Errors
+#### Presets
 
-Report errors are categorized by type:
+With `goodtables` different kind of tabular datasets could be validated. Tabular dataset is a something that could be split to list of data tables:
 
-- source - data can't be loaded or parsed
-- structure - general tabular errors like duplicate headers
-- schema - error of checks against JSON Table Schema
+![Dataset](https://raw.githubusercontent.com/frictionlessdata/goodtables-py/master/data/dataset.png)
 
-Report errors are categorized by context:
+To work with different kind of datasets we use `preset` argument for `validate` function. By default it will be inferred with `table` as a fallback value. Let's validate a [data package](http://specs.frictionlessdata.io/data-package/). As a result we get report of the same form but it will be having more that 1 table if there are more than 1 resource in data package:
 
-- table - the whole table errors like IO, HTTP or encoding error
-- head - headers errors
-- body - contents errors
+```py
+report = validate('datapackage.json') # implicit preset
+report = validate('datapackage.json', preset='datapackage') # explicit preset
+```
+
+To validate list of files we use `nested` preset. For nested preset first argument should be a list containing dictionaries with keys named after `validate` argument names. First argument is a `source` and we talk other arguments in next sections. Technically `goodtables` validates list of tables in parallel so it should be effective to do many tables validation in one run:
 
-### Presets
+```py
+report = validate([{'source': 'data1.csv'}, {'source': 'data2.csv'}]) # implicit preset
+report = validate([{'source': 'data1.csv'}, {'source': 'data2.csv'}], preset='nested') # explicit preset
+```
 
-Table is a main inspection object in goodtables. The simplest option is to pass to `Inspector.inspect` path and other options for one table (see example above). But when multitable parallized inspection is needed different presets could be used to process a dataset.
+#### Checks
 
-Let's see how to inspect a datapackage:
+Check is a main validation actor in goodtables. Every check is associated with a Data Quality Spec error. List of checks could be customized using `checks` argument. Let's explore options on an example:
 
 ```python
-from goodtables import Inspector
+report = validate('data.csv', checks='structure') # only structure checks
+report = validate('data.csv', checks='schema') # only schema checks
+report = validate('data.csv', checks={'bad-headers': False}) # exclude 'bad-headers'
+report = validate('data.csv', checks={'bad-headers': True}) # check only 'bad-headers'
+```
 
-inspector = Inspector()
-inspector.inspect('datapackage.json', preset='datapackage')
+By default a dataset will be validated against all available Data Quality Spec errors. Some checks could be not available for validation e.g. if schema is not provided only `structure` checks will be done.
+
+#### `validate(source, **options)`
```

Comment:
While reading the README I found this API documentation really confusing. It took me a while to understand that these are all params for the `validate` function but that depending on the preset different params are accepted. Perhaps instead of `[Dataset:table]` it would make more sense to use `[preset:table]`?